import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/backupuploader/LocalBackupUploadService.java

```diff
@@ -30,7 +31,10 @@ public void doWriteBackup(byte[] backupData, String uploadFilePath) throws Backu
     }
 
     @Override
-    public void removeBackup(String uploadFilePath) {
-        // todo: implement_me
+    public void removeBackup(String uploadFilePath) throws BackupExeption {
+        File file = new File(uploadFilePath);
+        if (!file.delete()){
```

Comment:
Better use here Files.delete() This method throws IOEx when we have any problems with processed file